tools/kmod/sof_remove.sh

```diff
@@ -156,7 +156,8 @@ remove_module snd_soc_intel_sof_maxim_common
 # SOF client drivers
 #-------------------------------------------
 remove_module snd_sof_probes
-remove_module snd_sof_ipc_test
+remove_module snd_sof_ipc_flood_test
```

Comment:
Please remove both the old and new name for the transition. Explain with a comment.

Reply:
I'm not following you, sorry.
`snd_sof_ipc_test` is no longer exist in my SOF client series as it was the flood test and it got renamed to `snd_sof_ipc_flood_test`

Since we have a new debug feature for injecting messages, we also have a new client driver: `snd_sof_ipc_msg_injector`

The commit message tells this also.

Reply:
> snd_sof_ipc_test is no longer exist in my SOF client series

sof-test is used to test older sof branches.

Assuming we'd even want that[*], we don't have a way to "atomically" upgrade both sof and sof-test at the same time for everyone.

[*] good luck bisecting a test failure then.

Reply:
OK, I'll keep the retired module name.
Just a sec.

Reply:
I'm happy with the code change now but the comment does not seem to hint at backward compatibility at all. So I'm afraid someone not well informed and trigger-happy could just go and remove that line

Reply:
Or just drop the comment ;)

Old PRs with the `snd_sof_ipc_test` do not apply to sof-dev anymore due to extensive changes and it is not present in my updated PR, but it might re-surface.

Reply:
Again, why not just add this latest and greatest explanation as a comment? It's not like this script is drowning under too many comments and having to lookup and go to the code review to find this sort of information is tedious and uncertain.

Reply:
I don't think it is good to have multi-line comment for a module name which was used during the development cycle of a big feature set.
I figured that instead of going word by word changes in a comment, it is cleaner to have no comment at all.

Reply:
> which was used during the development cycle of a big feature set.

Sorry I missed the "old" one was never merged in the first place.